lib/Doctrine/DBAL/Schema/AbstractSchemaManager.php

```diff
@@ -917,7 +917,7 @@ protected function _getPortableTablesList($tables)
     /**
      * @param array $table
      *
-     * @return array
+     * @return mixed[]|string
```

Comment:
Can elements of the array be anything else than string? `string[]|string` would make more sense.

Reply:
No idea since `$table` is uselessly types as `array`. If you are certain, I can change it.

Reply:
I'd look at it from the caller's standpoint: whatever callers can consume by design (`string[]` or `string`) should be valid return, the rest shouldn't.

Reply:
Please figure out the correct type from the API standpoint and ignore the deviations.

Reply:
```
 ------ ------------------------------------------------------------------------------------------------------------------------ 
  Line   lib/Doctrine/DBAL/Schema/OracleSchemaManager.php                                                                        
 ------ ------------------------------------------------------------------------------------------------------------------------ 
  104    Method Doctrine\DBAL\Schema\OracleSchemaManager::_getPortableTableDefinition() should return array but returns string.  
 ------ ------------------------------------------------------------------------------------------------------------------------ 

 ------ ---------------------------------------------------------------------------------------------------------------------------- 
  Line   lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php                                                                        
 ------ ---------------------------------------------------------------------------------------------------------------------------- 
  222    Method Doctrine\DBAL\Schema\PostgreSqlSchemaManager::_getPortableTableDefinition() should return array but returns string.  
 ------ ---------------------------------------------------------------------------------------------------------------------------- 

 ------ --------------------------------------------------------------------------------------------------------------------------- 
  Line   lib/Doctrine/DBAL/Schema/SQLServerSchemaManager.php                                                                        
 ------ --------------------------------------------------------------------------------------------------------------------------- 
  210    Method Doctrine\DBAL\Schema\SQLServerSchemaManager::_getPortableTableDefinition() should return array but returns string.
```
Honestly, no idea. Some managers return `$table` as is and some return a string (schema name + table name).
This definitely looks like a bad design (git-blame indicates addition back in 2008-2011).
Help? :)

Reply:
putting it into ignores for now, unlikely to be properly fixable in 2.x